### PR TITLE
Unzip alias

### PR DESCRIPTION
**Pull request by Redth**
This is a simple method to Unzip a specified file to a given directory.

### CODE REVIEW

File: src/Cake.Common/IO/Zipper.cs [hunk 1]
```diff
@@ -102,6 +102,34 @@ public void Zip(DirectoryPath rootPath, FilePath outputPath, IEnumerable<FilePat
             _log.Verbose("Zip successfully created: {0}", outputPath.FullPath);
         }
 
+        /// <summary>
+        /// Unzips the specified file to the specified output path
+        /// </summary>
+        /// <param name="zipPath">Zip file path.</param>
+        /// <param name="outputPath">Output directory path.</param>
+        public void Unzip(FilePath zipPath, DirectoryPath outputPath)
+        {
```
Comment (devlead) [inline src/Cake.Common/IO/Zipper.cs]: Minor, but a bunch of white space here.

File: src/Cake.Common/IO/Zipper.cs [hunk 2]
```diff
@@ -102,6 +102,34 @@ public void Zip(DirectoryPath rootPath, FilePath outputPath, IEnumerable<FilePat
...
+            outputPath = outputPath.MakeAbsolute(_environment);
+
+            // Open up a stream to the output file.
+            _log.Verbose("Unzipping file: {0}", zipPath.FullPath);
+
+            ZipFile.ExtractToDirectory(zipPath.FullPath, outputPath.FullPath);
+
+            _log.Verbose("Unzipped to: {0}", outputPath.FullPath);
+        }
+
```
Comment (devlead) [inline src/Cake.Common/IO/Zipper.cs]: Minor, but a bunch of white space here.

**Comment by devlead** [comment]
@Redth Looks good, I'll take a look later to day and merge if I don't find anything else.

File: src/Cake.Common/IO/Zipper.cs (hunk 1 above)
Comment (Redth) [inline src/Cake.Common/IO/Zipper.cs]: Removed some whitespace

**Pull request merged**